apps/wallet-mobile/src/components/Warning/Warning.tsx

```diff
@@ -8,14 +8,21 @@ import {Space} from '../Space/Space'
 type Props = {
   content: ReactNode
   iconSize?: number
+  title?: string
 }
 
-export const Warning = ({content, iconSize = 30}: Props) => {
+export const Warning = ({content, iconSize = 30, title}: Props) => {
```

Comment:
```suggestion
export const Warning = ({content, iconSize = 30, title = ""}: Props) => {
```

just for consistency

Reply:
applied